builtin/providers/openstack/resource_openstack_compute_secgroup_v2.go

```diff
@@ -210,10 +212,20 @@ func resourceComputeSecGroupV2Delete(d *schema.ResourceData, meta interface{}) e
 		return fmt.Errorf("Error creating OpenStack compute client: %s", err)
```

Comment:
Noticed this message should be `s/creating/deleting/` while reviewing the diff below.